types/src/header.rs

```diff
@@ -50,6 +51,8 @@ pub struct Header {
     transactions_root: H256,
     /// State root.
     state_root: H256,
+    /// Validator set root. Note that this is for the next block.
+    validator_set_root: H256,
```

Comment:
Why did you choose the term "root"?
Is the validator set is stored in a tree?

Reply:
What do you think about adding the "next_" prefix to the field name?
When I first read the "validator_set_root", it seems that it is the current block's validator set.

Reply:
Ok then I'll rename it as `next_validator_set_hash` after your overall review